tinker/configuration.py

```diff
@@ -5,6 +5,8 @@
 from typing_extensions import TypedDict
```

Comment:
I believe it's part of the `mypy` package. I'm not sure what the bug here would be, but if there truly is a bug, please file an issue.

Reply:
It's a dependency of `mypy` at least. The bug is that when this is installed without `mypy` (e.g. from a PYPI pip install in the future), this will raise an ImportError. Unless typing extensions is also an "install_requires" entry for `tinker`. I will file an issue.

Reply:
Please link that issue here after you file it; then I can resolve this thread. Thanks!

Reply:
I've created #137